packages/ti_eset/manifest.yml

```
@@ -0,0 +1,57 @@
+format_version: 2.13.0
+name: ti_eset
+title: "ESET Threat Intelligence"
+version: 0.1.0
+source:
+  license: "Elastic-2.0"
+description: "Ingest threat intelligence indicators from ESET Threat Intelligence with Elastic Agent."
+type: integration
+categories:
+  - security
+  - threat_intel
+conditions:
+  kibana:
+    version: "^8.10.1"
+  elastic:
+    subscription: "basic"
+screenshots:
+  - src: /img/overview.png
+    title: threat intelligence overview
+    size: 600x600
+    type: image/png
+  - src: /img/files.png
+    title: threat intelligence files
+    size: 600x600
+    type: image/png
+  - src: /img/url-ip.png
+    title: threat intelligence domains, URLs and IPs
+    size: 600x600
+    type: image/png
+icons:
+  - src: /img/eset-lozenge-color-rgb.svg
+    title: Sample logo
+    size: 96x96
+    type: image/svg+xml
+policy_templates:
+  - name: eset
+    title: ETI feeds (TAXII version 2)
+    description: Collect data from ETI feeds (TAXII version 2)
+    inputs:
+      - type: httpjson
+        title: ETI feeds (TAXII version 2)
+        description: Collect data from ETI feeds (TAXII version 2)
+        vars:
+          - name: username
+            type: text
+            title: ETI Username
+            required: true
+            show_user: true
+          - name: password
+            type: password
+            title: ETI Password
+            multi: false
+            required: true
+            show_user: true
+owner:
+  github: elastic/integrations
```

```suggestion
  github: elastic/security-service-integrations
```
CI is failing because the owner doesn't match `.github/CODEOWNERS`